### DISCUSSION TITLE
Allow to override or append the Ctrl+Space shortcut

### DISCUSSION THREAD

**Discussion by plimarenko** [tprouvot/Salesforce-Inspector-reloaded#418] [Ideas]
On Mac, the **`Ctrl`+`Space`** shortcut is by default used for the **Keyboard Input Source** selection, hence the multi-language users cannot use the **Select all fields in a query** feature w/o having to do remapping, which is also messes up with the muscle memory :-)
It would be really great to allow users to override the shortcut with a custom one or at least to be able to add an extra key, e.g. **`Ctrl`+`Space`+`A`**

**Comment by tprouvot** [comment]
Hi @plimarenko,
I did [something related to shortcuts ](https://github.com/tprouvot/Salesforce-Inspector-reloaded/issues/171) customization, I'll check if I Can leverage this for Ctrl + Space

**Reply by IamKewal** [reply]
Also It would be great to add a shortcut to select the first suggestion in the Data export using tab or something. 
Like If I am writing a query, I have to make transitions from keyboard to mouse each time to select any suggestion. 

Or to be even more productive, shortcut can be like Alt + 1 to select first suggestion, Alt + 2 to select second and so on.

**Reply by tprouvot** [reply]
Hi @IamKewal ,
Today you can already iterate through values with tab key and select it with enter.

**Comment by tprouvot** [comment]
Hi @plimarenko,
This has been developed and will be release in the 1.26 and in the beta version mid Jan 25 ! 
More details here: 
https://github.com/tprouvot/Salesforce-Inspector-reloaded/issues/653
